Automattic/wc-calypso-bridge
# Deploy Automation: Add bump version command

### Changes proposed in this Pull Request:

Adds a new command to bump the version and updates the `composer.json` and  `wc-calypso-bridge.php` files.

Closes #1097
Related to #1096


### How to test the changes in this Pull Request:


1. Clone this branch.
2. In the ./scripts directory, run `npm install` to manually install the script dependencies.
3. Run `npm run bump-version`
4. Select a change degree
![Screen Shot 2023-05-03 at 01 16 54](https://user-images.githubusercontent.com/1234758/235832027-775eb8cc-9b5b-4a34-a983-60ca4af292ec.png)

5. Confirm the new version

![Screen Shot 2023-05-04 at 16 16 21](https://user-images.githubusercontent.com/1234758/236306410-9fc85ceb-8ed4-4fe8-a3c3-bb1b933e523e.png)

6. Check the `composer.json` file and verify that it shows the version on the `version` value.
![Screen Shot 2023-05-03 at 01 18 03](https://user-images.githubusercontent.com/1234758/235832119-8796450d-5ac1-4d50-ad6a-69da311eb24a.png)

7. Check the `wc-calypso-bridge.php` file and verify that version in the comment and the constant were updated:
![Screen Shot 2023-05-03 at 01 19 52](https://user-images.githubusercontent.com/1234758/235832306-ca98f47c-3764-445b-bafe-70a52acb2b9d.png)
![Screen Shot 2023-05-03 at 01 19 57](https://user-images.githubusercontent.com/1234758/235832293-224fe818-ef69-4a7d-9c51-2ea24eca6ee2.png)

8. Verify that a new branch named `update-version-VERSION` was created and use `git log` to verify that the changes were committed:
![Screen Shot 2023-05-04 at 16 18 17](https://user-images.githubusercontent.com/1234758/236306922-7d2856d1-9d53-46fb-943c-d1b7f37d8145.png)

9. Checkout to other branch and delete the test branch you just created:
```
git branch -D update-version-x.x.x
```





### Other information:




### FOR PR REVIEWER ONLY:
